config/filter.d/apache-pass.conf

```diff
@@ -0,0 +1,20 @@
+# Fail2Ban Apache pass filter
+# This filter is for access.log, NOT for error.log
+#
+# The knocking request must have a referer.
+
+[INCLUDES]
+
+before = apache-common.conf
+
+[Definition]
+
+failregex = ^<HOST> - \w+ \[\] "GET <knocking_url> HTTP/1\.[01]" 200 \d+ ".*" "[^-].*"$
+
+ignoreregex =
+
+[Init]
+
+knocking_url = /knocking/
```

Comment:
But you DO define it: https://github.com/fail2ban/fail2ban/pull/1112#issuecomment-121317700

You can't have it both ways...take issue with a comment about defining it and then define it, saying you weren't supposed to define it....while defining it anyway....